src/components/ExtensionRoot.js

```diff
@@ -1,16 +1,17 @@
 import { AuthProvider } from "@aha-app/aha-develop-react";
```

Comment:
git has picked this up as a file rename, but actually this file moved to src/views/prsPage.js and the ExtensionRoot is an extraction from all the react entry points.